.github/workflows/test-deploy-main.yml

```diff
@@ -167,3 +167,33 @@ jobs:
       - name: Run yarn build on each sample app
         run: yarn build
         working-directory: ./canary
+
+  trigger-failure-alarm:
+    # Triggers an alarm if any of builds failed.
+    runs-on: ubuntu-latest
+    environment: ci
+    needs: build
+    if: ${{ failure() }}
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - run: aws cloudwatch put-metric-data --metric-name publishFailure --namespace GithubCanaryApps --value 1
+
+  trigger-success-alarm:
+    # Triggers an alarm if all the builds succeed
+    runs-on: ubuntu-latest
+    environment: ci
+    needs: build
+    if: ${{ success() }}
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - run: aws cloudwatch put-metric-data --metric-name publishSuccess --namespace GithubCanaryApps --value 0
```

Comment:
Same here with these two alarms.

Reply:
Same. I think this line is only for metrics

Reply:
We configured canary to trigger alarm on missing data. 

I don't think we can do that here though, because publish to `@next` does not happen on a regular schedule. IMO should have `@next` trigger alarm only on failure data and ignore missing data.

Reply:
Right, my thought here is that we shouldn't even need to add success metrics here. We can safely delete `trigger-success-alarm` and only have `trigger-failure-alarm` job.